sdk/resourcemanager/azure-resourcemanager-servicebus/src/main/java/com/azure/resourcemanager/servicebus/fluent/inner/QueueResourceInner.java

```diff
@@ -0,0 +1,605 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.resourcemanager.servicebus.fluent.inner;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.annotation.JsonFlatten;
+import com.azure.core.management.Resource;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.resourcemanager.servicebus.models.EntityAvailabilityStatus;
+import com.azure.resourcemanager.servicebus.models.EntityStatus;
+import com.azure.resourcemanager.servicebus.models.MessageCountDetails;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.OffsetDateTime;
+
+/** The QueueResource model. */
+@JsonFlatten
+@Fluent
+public class QueueResourceInner extends Resource {
```

Comment:
Manual changes from `extends ProxyResource` to `extends Resource` as the Impl class requires `<? extends Resource>`. The tags of Resource would not expose to end users by restricting fluent interfaces stages.